cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -111,9 +111,10 @@
   static final String NAMESPACE_CPU_LIMIT_PROPERTY = "k8s.namespace.cpu.limits";
   static final String NAMESPACE_MEMORY_LIMIT_PROPERTY = "k8s.namespace.memory.limits";
 
+  public static final String SECURITY_CONFIG_NAME = "cdap-security";
```

Comment:
I think we may run into problems if we hardcode the `cdap-security` volume mount name as it is actually specified in the CDAP CR, so we may want to make this configurable.

Reply:
Ack